frontend/src/metabase/query_builder/components/EditableText/EditableText.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+
+import { Root, BorderedInput } from "./EditableText.styled";
+
+import { KEY_ESCAPE } from "metabase/lib/keyboard";
+
+interface Props {
+  value: string;
+  onChange?: (val: string) => void;
+}
+
+export const EditableText = ({ value, onChange }: Props) => {
+  const [memo, setMemo] = useState(value);
+
+  const handleChange = (e: React.ChangeEvent) => {
+    const target = e.target as HTMLInputElement;
```

Comment:
I think you should be able to type the event without having to cast it later
```suggestion
  const handleChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
    const target = e.target;
```